src/components/terms-and-conditions.tsx

```diff
@@ -0,0 +1,205 @@
+// Terms and Conditions Pop-up
+
+import React, { ReactNode, useState } from "react";
+import { Inter } from "next/font/google";
+import { Controller, SubmitHandler, useForm } from "react-hook-form";
+
+const inter = Inter({ subsets: ["latin"] });
+
+type FormValues = {
+  tncAccepted: false;
+};
```

Comment:
when declaring Typescript types, you should declare them with actual data types.  Here false should instead be boolean